core/src/test/scala/org/apache/spark/deploy/SparkSubmitSuite.scala

```diff
@@ -1609,6 +1609,16 @@ class TestFileSystem extends org.apache.hadoop.fs.LocalFileSystem {
   }
 
   override def open(path: Path): FSDataInputStream = super.open(local(path))
+
+  // No-op methods
+
+  override def copyFromLocalFile(
+    delSrc: Boolean,
+    overwrite: Boolean,
+    src: Path,
+    dst: Path): Unit = {}
+
+  override def mkdirs(path: Path): Boolean = true
```

Comment:
Please revert this change and make a new test file system in `BasicDriverFeatureStepSuite.scala` and use it.